### PR TITLE
Add Aaron (Arch Linux user btw) to yearbook

### PR DESCRIPTION
**Pull request by Arinerron**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [ ] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by Arinerron** [comment]
@juanpflores I'm certain I filled it out. Is it case--sensitive? I probably used the username `arinerron` instead of `Arinerron`. If you still don't see it, I'll fill it out again. Thanks!

**Comment by Arinerron** [comment]
Hi @elisemoe!

I filled out the form again (& took screenshot for proof):

image

Let me know if you get it.

File: _data/arinerron/arinerron.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Aaron Esau
+institution: Tigard High School
+profile_pic: arinerron2-filled.png	
+quote: i use arch btw '>"><svg/onload=alert()> ${{2*2}}
```
Comment (juanpflores) [inline _data/arinerron/arinerron.md:5]: Are you sure you want to push this 🤔

Comment (Arinerron) [reply _data/arinerron/arinerron.md]: Alright, alright, updated the quote in https://github.com/Arinerron/graduation/commit/f1b3a24a28de1fdd2165a0435d5340baba24ad08

File: _data/arinerron/arinerron.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Aaron Esau
+institution: Tigard High School
+profile_pic: arinerron2-filled.png	
+quote: "If you think your users are idiots, only idiots will use it." -Linus Torvalds
```
Comment (juanpflores) [inline _data/arinerron/arinerron.md:5]:
```suggestion
quote: If you think your users are idiots, only idiots will use it. -Linus Torvalds
```

**Pull request closed without merging**